saleor/graphql/checkout/mutations.py

```diff
@@ -440,16 +447,28 @@ class Meta:
         error_type_field = "checkout_errors"
 
     @classmethod
-    def validate_checkout_lines(cls, variants, quantities, country, channel_slug):
-        check_lines_quantity(variants, quantities, country, channel_slug)
+    def validate_checkout_lines(
```

Comment:
Question -  Do we really need this method? It looks like it just run one function which can be executed inside a clean_input. IMHO I don't see any prons of having this here. 
cc @maarcingebala

Reply:
If I'm correct we need to have control over arguments passed to `check_lines_quantity` in `CheckoutLinesUpdate` mutation that inherits from `CheckoutLinesAdd`.

Reply:
Oh, right. I didn't think about this. Thanks! 👨‍🏫